README.md

```diff
@@ -65,7 +81,7 @@ To compile the project yourself:
     
 2. Copy the newly created `.aar` and `.gdap` files to your plugin directory:
 
-    `app/build/outputs/aar/GodotGooglePlayBilling-6.X.X-release.aar` to `[your godot project]/android/plugins/`
```

Comment:
`from `app/build/outputs/aar/GodotGooglePlayBilling-6.X.X-release.aar` to `[your godot project]/android/plugins/``